src/controllers/answer.js

```diff
@@ -0,0 +1,36 @@
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "next" }] */
+import data from '../data.json';
+
+/** Class representing a question. */
```

Comment:
Document your classes using JSDoc format too. You can find how to do that in their documentation